django_ipam/api/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^subnet/(?P<subnet_id>[0-9a-f-]+)/iprequest/$', views.iprequest, name='iprequest'),
```

Comment:
use the same regexp pattern used here: https://github.com/netjson/django-netjsongraph/blob/master/django_netjsongraph/utils.py#L47-L49